feature-libs/quote/components/details/comment/quote-details-comment.component.ts

```diff
@@ -70,6 +77,22 @@ export class QuoteDetailsCommentComponent {
       );
   }
 
+  onItemClicked(event: { item: Item }) {
+    this.quoteDetailsCartService.setQuoteEntriesExpanded(true);
+    this.quoteDetailsCartService
+      .getQuoteEntriesExpanded()
+      .pipe(take(1), delay(0)) // delay this task until the UI has expanded
+      .subscribe(() => {
+        const aTags = this.document.getElementsByTagName('a');
```

Comment:
```suggestion
       const aTags = Array.from(this.document.getElementsByTagName('a'));
```
@Uli-Tiger 
Then you can avoid the sonar issue